pyproject.toml

```diff
@@ -31,6 +31,9 @@ filterwarnings = [
 
     # Warnings in test_othermlpackages
     "default:The LGBMClassifier or classes from which it inherits use `_get_tags` and `_more_tags`:FutureWarning",
+
+    # Warnings in scipy.optimize due to incompatibility with sklearn
```

Comment:
Does it appear just for the #all-deps errors? If so, I would place it right under the rest from that category. Otherwise we can keep it as it is here, if it is appearing in more builds.